oceanprotocol/ocean-node
# Dashboard: display index queue

Fixes #290 

Changes proposed in this PR:

- The dashboard now shows the indexing queue: 
![image](https://github.com/oceanprotocol/ocean-node/assets/20739535/f8d6e33c-9a20-4fe3-a4da-3e048e9bb4ad)


### To Test:

1. `npm run build && npm run start`

2. Add a transaction to the indexing queue. send a POST request to the direct command endpoint: `http://127.0.0.1:8000/directCommand`

Example body:

```
{
    "command": "reIndex",
    "chainId": 137,
    "txId": "0xc53ba79bdcdba60c466c401631edc97383c230e1fdd1a734737cdab033446923"
}
```

3. You should see the transaction appear in the indexing queue on the dashboard.
